pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -320,7 +320,9 @@ def solve(self, model, **kwds):
 
             MindtPy = solve_data.working_model.MindtPy_utils
             setup_results_object(solve_data, config)
-            process_objective(solve_data, config, use_mcpp=False)
+
+            # Process objective in case that it is nonlinear move it to constraints
+            process_objective(solve_data, config)
```

Comment:
We didn't notice that the merge removed the option `use_mcpp=False`, from this function. Please fix it.